src/services/lnd/onchain-service.ts

```diff
@@ -104,3 +136,23 @@ export const extractIncomingTransactions = (
         }),
     )
 }
+
+export const extractOutgoingTransactions = ({
```

Comment:
shoundn't this belong to domain?

Reply:
this methods are like `userFromRaw`, `translateToLedgerTx` or `userIPsFromRaw` (internal implementation of service) the difference here is that are exported  to do unit testing.

Reply:
there are some logic in here with the `filter`, so in my mind, this is not just a mapping

Reply:
that could be the case if we add a generic listTransactions in domain interface but we only have `listIncomingTransactions` src/domain/bitcoin/onchain/index.types.d.ts (how that is implemented is responsability of the service).

Reply:
probably not really needed as of now